.storybook/blocks/Table/Table.module.css

```diff
@@ -0,0 +1,17 @@
+.table {
+  width: 100%;
+  border-collapse: collapse;
+}
+
+.table th, .table td {
+  text-align: left;
+  padding: 8px;
+}
+
+.table, .table th, .table td {
+  border: 1px solid var(--color-mica-blue-200);
+}
```

Comment:
available to use `nesting` css
```
.table {
  th,td {}
}
```

Reply:
I think this kind of nesting is only available in SASS and LESS. I used the `&` operator instead

Reply:
cool, thanks